packages/api-client/src/libs/create-client.ts

```diff
@@ -333,5 +338,26 @@ export const createApiClient = ({
     modalState,
     /* The workspace store */
     store,
+    /** Update the currently selected example */
+    updateExample: (exampleKey: string, operationId: string) => {
+      if (!exampleKey || !operationId) return
```

Comment:
this is also good enough for now, but if requested we can also handle cases of no operationId

Reply:
noted! it is one way to retrieve the request but can also be done differently